portfolio/src/main/webapp/script.js

```diff
@@ -73,14 +73,14 @@ function storeNumberComments() {
 
 function getServerComments() {
   fetch('/data?number-comments='+number).then(response => response.json()).then((commentSection) => {
-    if (commentSection.length == 0 && number != 0){
-      displayTotalComments(commentSection);
-    } else if (commentSection.length > 0) {
+    if (commentSection.length < number){
       displayTotalComments(commentSection);
       populateComments(commentSection);
-    } else {
+    } else if (number == 0) {
```

Comment:
Seems like this is from a previous pull request, but `number` as a variable name doesn't really tell me anything about what this value means/is used for. It could use a more descriptive name.

Reply:
Additional FYI, `==` and `===` mean different things in JavaScript. I would encourage you to look into that and make sure you're using the one you think most appropriate here.